djangocms_admin_style/sass/djangocms-admin.scss

```diff
@@ -1,4 +1,5 @@
-@import "_settings.scss";
+@import "settings/_settings.scss";
```

Comment:
we should probably name this one `_cms.scss` and have `_all.scss` in settings folder so we can reference them from there